src/python/pants/engine/fs.py

```diff
@@ -271,7 +271,7 @@ def write_digest(self, digest: Digest, *, path_prefix: Optional[str] = None) ->
 
 
 _EMPTY_FINGERPRINT = "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
-EMPTY_DIGEST = Digest(fingerprint=_EMPTY_FINGERPRINT, serialized_bytes_length=0)
+EMPTY_DIGEST: Digest = Digest(fingerprint=_EMPTY_FINGERPRINT, serialized_bytes_length=0)
```

Comment:
Not sure why MyPy wanted me to do this. It complained in a callsite that it couldn't figure out the type for `EMPTY_DIGEST`.